### DISCUSSION TITLE
How to add a custom CSS class?

### DISCUSSION THREAD

**Discussion by fmariofilho** [slidevjs/slidev#1468] [Q&A]
I would like to add a custom CSS class that I can use in all my slides.  Something like:

```
.myclass {
    background-color: red;
}
```

From what I understood of the documentation, I added this to a file called `style.css` in the same directory as my `slides.md` file.  Then in my slide I want to do:

```
<div class="myclass">
etc
</div>
```

It doesn't work.  What am I doing wrong?

**Comment by kermanx** [comment]
The code looks fine. How did you run Slidev?

**Reply by fmariofilho** [reply]
I have a global install of slidev and I'm running `npx slidev` from inside the folder with `slides.md`.  The funny thing is that if I make an `index.html` file and put the CSS code there, it works.

**Reply by kermanx** [reply]
I can't reproduce the problem on my machine. You can provide a minimal reproduction via https://sli.dev/new if you want.
